provision/kubernetes/job.go

```diff
@@ -29,6 +31,7 @@ import (
 const (
 	promNamespace = "tsuru"
 	promSubsystem = "job"
+	expireTTL     = time.Hour * 24 // 1 day
```

Comment:
@arthurcgc I prefer to have two variables to cause curiosity of code reader to understand logic bellow.

```
frequentExpireTTL = time.Hour * 24
occasionalExpireTTL = time.Hour
```